sceptre/cli.py

```diff
@@ -235,6 +235,22 @@ def describe_stack_resources(ctx, environment, stack):
     write(response, ctx.obj["output_format"])
 
 
+@cli.command(name="diff-stack")
+@stack_options
+@click.pass_context
+@catch_exceptions
+def diff_stack(ctx, environment, stack):
+    """
+    Shows the local template differences against cloudformation.
+
+    Prints differences.
+    """
```

Comment:
This doc string should be more informative. Something to the extent of, "Print the diff between the currently deployed stack with the local version of that stack" would be better, in my opinion.